### PR TITLE
Commits cherry-picked from the release 0.8 branch

### PR DESCRIPTION
**Pull request by mbalassi**
These commits have only been pushed to the release-0.8 branch, but to my understanding are also needed on the master.

They contain some minor fixes, dependency, readme and licence clean ups.

### CODE REVIEW

**Comment by rmetzger** [comment]
Hi,
can you remove the commit "[FLINK-1385] Print warning if not resources are _currently_ available… …" from the pull request?
Since I have a major YARN client rewrite pull request pending, I'll include a fix for the rewrite into the pull request.

**Comment by mbalassi** [comment]
Thanks for the heads up, done.

File: docs/faq.md [hunk 1]
```diff
@@ -102,6 +102,11 @@ the source of the error. Solutions are to
 - make the enclosing class serializable
 - use a Java 8 lambda function.
 
+### In Scala API, I get an error about implicit values and evidence parameters
```
Comment (StephanEwen) [inline docs/faq.md]: I made a typo here. Can you change it to `In the Scala API`?

File: pom.xml [hunk 2]
```diff
@@ -32,9 +32,9 @@ under the License.
 	<artifactId>flink-parent</artifactId>
 	<version>0.9-SNAPSHOT</version>
 
-	<name>flink</name>
+	<name>Apache Flink</name>
```
Comment (StephanEwen) [inline pom.xml]: I vote to undo this change. This is the name displayed in maven (command line and more) and not just some "pretty id" as I thought.

File: LICENSE [hunk 3]
```diff
@@ -218,16 +218,18 @@ conditions of their respective licenses.
...
  - Sizzle.js (http://sizzlejs.com) - Copyright 2010, The Dojo Foundation
  - jCanvas 13.11.21 (http://calebevans.me/projects/jcanvas/) - Copyright 2014 Caleb Evans
  - Flot 0.8.1 (http://www.flotcharts.org) - Copyright (c) 2007-2013 IOLA and Ole Laursen
+ - query.flot.tooltip (https://github.com/krzysu/flot.tooltip) - Copyright (c) Krzysztof Urbas
  - Bootstrap 3.1.0 (http://getbootstrap.com) - Copyright (c) 2011-2014 Twitter, Inc
+ - Bootswatch Lumen Theme v3.1.1+1 (http://bootswatch.com) - Copyright (c) 2012-2014 Thomas Park
  - normalize.css v3.0.0 (http://git.io/normalize) - Copyright (c) Nicolas Gallagher and Jonathan Neal
  - Font Awesome - Code (http://fortawesome.github.io/Font-Awesome/) - Copyright (c) 2014 Dave Gandy
  - D3 dagre renderer (https://github.com/cpettitt/dagre-d3) - Copyright (c) 2012-2013 Chris Pettitt
  - scopt (http://github.com/scopt/scopt)
```
Comment (StephanEwen) [inline LICENSE]: This LICENSE file is only for dependencies that we ship with the source (like css and js and fonts and copy/pasted third party source code under for example BSD and MIT license)

Remove the `scopt` line for that reason

File: flink-dist/src/main/flink-bin/LICENSE [hunk 4]
```diff
@@ -249,13 +252,17 @@ The Apache Flink project bundles the following components under the MIT License:
...
  - Sizzle.js (http://sizzlejs.com) - Copyright 2010, The Dojo Foundation
  - jCanvas 13.11.21 (http://calebevans.me/projects/jcanvas/) - Copyright 2014 Caleb Evans
  - Flot 0.8.1 (http://www.flotcharts.org) - Copyright (c) 2007-2013 IOLA and Ole Laursen
+ - query.flot.tooltip (https://github.com/krzysu/flot.tooltip) - Copyright (c) Krzysztof Urbas
  - Bootstrap 3.1.0 (http://getbootstrap.com) - Copyright (c) 2011-2014 Twitter, Inc
+ - Bootswatch Lumen Theme v3.1.1+1 (http://bootswatch.com) - Copyright (c) 2012-2014 Thomas Park
  - normalize.css v3.0.0 (http://git.io/normalize) - Copyright (c) Nicolas Gallagher and Jonathan Neal
  - Font Awesome - Code (http://fortawesome.github.io/Font-Awesome/) - Copyright (c) 2014 Dave Gandy
  - D3 dagre renderer (https://github.com/cpettitt/dagre-d3) - Copyright (c) 2012-2013 Chris Pettitt
+ - spymemcached (http://code.google.com/p/spymemcached/) - Copyright (c) 2006-2009  Dustin Sallings, Copyright (c) 2009-2011  Couchbase, Inc.
```
Comment (StephanEwen) [inline flink-dist/src/main/flink-bin/LICENSE]: Add here ` - scopt (http://github.com/scopt/scopt) - Copyright (c) scopt contributors`

**Comment by StephanEwen** [comment]
I made some comments inline. Otherwise it looks good.

Thanks for work!

**Comment by StephanEwen** [comment]
I addressed the comments and committed the changes to the 0.9 master. I forgot to add the "This closes #298" message, thought. Marton, can you close this Pull Request manually?

**Comment by mbalassi** [comment]
Thanks, done.
